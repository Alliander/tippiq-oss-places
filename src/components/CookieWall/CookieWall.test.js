import React from 'react';
import renderer from 'react-test-renderer';
import Element from './CookieWall';

describe('CookieWall', () => {
  test('Renders', () => {
    const component = renderer.create(
      <Element>
        <p>Text<a>Link</a></p>
        <button>Akkoord</button>
      </Element>
    );
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
