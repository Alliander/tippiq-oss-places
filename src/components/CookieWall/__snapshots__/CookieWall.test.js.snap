// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CookieWall Renders 1`] = `
<div
  className={undefined}
>
  <p
    className={undefined}
  >
    Tippiq gebruikt Cookies.
     
    <a
      className={undefined}
      href={undefined}
      rel="noopener noreferrer"
      target="_blank"
    >
      Lees hier waarom.
    </a>
     
    Als je doorgaat, accepteer je het gebruik van deze cookies.
  </p>
  <button
    className={undefined}
    id="btnAgree"
    onClick={undefined}
  >
    Ik ga akkoord
  </button>
</div>
`;
